jenkins/bin/cat_envs.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
@jgmize @glogiotatidis thought about splitting this out into a separate small library as I'm guessing other projects might want to use it. Thoughts?